packages/go_router/lib/src/match.dart

```diff
@@ -79,6 +82,9 @@ class RouteMatch {
   /// Query parameters for the matched route.
   final Map<String, String> queryParams;
 
+  /// Query parameters for the matched route.
```

Comment:
Can you add a bit more explanation on whats different between this and queryParams?

Reply:
I tried to write a better documentation in https://github.com/flutter/packages/pull/2392/commits/d9f93151e6ff9092ddace07be1ed450f5ab8b112

Tell me if there is something to change